### ISSUE TITLE
Make voice/mobile experience better

### ISSUE THREAD

**Issue by tarasglek** [tarasglek/chatcraft.org#309]
1. Current mic input on mobile is buggy. I'm not sure if that's because of the ui that requires holding small button down or some other failure.
2. Would be nice to be able to use voice for input when editing system prompt and messages.
3. It's trivial to use on-device TTS. We should add a speak button/checkbox/modality. See example https://chatcraft.org/c/tarasglek/xpvh4QZzMeZW-ms0VsCgL This makes it more fun to use chatcraft in a social setting.
4. It would be nice to have serverside profiles, it's really sad to not have a collection of system prompts on mobile.

Use-case:
Surprisingly I use chatcraft a lot in the car, usually as a passenger. This will be especially useful once #285 lands. Usually I change the prompt to be related to something I'm being driven by or some thought that came into my bored head and then use voice to input my question and talk about that subject. With vision I'll be able to snap nearby cars and ask about specs, prices, etc or of interesting landmarks.
